doc/mergerepo_c.8

```diff
@@ -1,6 +1,6 @@
 .\" Man page generated from reStructuredText.
 .
-.TH MERGEREPO_C  "2019-07-18" "" ""
+.TH MERGEREPO_C  "2020-04-29" "" ""
```

Comment:
What's the date for?

Reply:
I think is supposed to be `The date of the last nontrivial change that was made to the man page.` from [here](http://man7.org/linux/man-pages/man7/man-pages.7.html). But we use this [script](https://github.com/rpm-software-management/createrepo_c/blob/master/utils/gen_rst.py) to generate the man pages from GOptionEntry structs in the code and the script just updates the date. 

Its not perfect because there are some more detailed descriptions in the man pages as opposed to the GOptionEntries that the script overrides, I usually just don't include them in the commit but in general the script is helpful.

The script was probably used to or was meant to generate the man pages at rpm build time.

Reply:
I see, thanks. So just on a side note if the man page is actually generated then it shouldn't be stored in git, it should be generated by the build system - unless there's a reason, but then that's probably still a bad design that should be fixed in a different way... :sunglasses: